core/modules/user/store/actions.ts

```diff
@@ -43,6 +44,17 @@ const actions: ActionTree<UserState, RootState> = {
       Vue.prototype.$bus.$emit('session-after-started')
     })
   },
+  /**
+   * Retrieve authenticated user from browser cache synchronously
+   */
+  loadUserEarly (context) {
```

Comment:
i guess it should be named `loadFromCache` or sth similar

Reply:
we currently have this convention as just a load function. can't we deal with the cache in there?

Reply:
sure, renamed